propagator/ottrace/lib/opentelemetry/propagator/ottrace/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Propagator
+    module OTTrace
+      VERSION = '0.1.0'
```

Comment:
```suggestion
      VERSION = '0.15.0'
```

We're generally keeping versions (mostly) in lockstep at the moment.

Reply:
_That is what we in the industry refer to as a "typo"_ 🤦🏼